src/main/java/task63/NumberOfCharacters.java

```diff
@@ -0,0 +1,25 @@
+package task63;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class NumberOfCharacters {
+    public static void main(String[] args) throws FileNotFoundException {
```

Comment:
Analogicznie jak w poprzednim zadaniu.

Chcielibyśmy, aby nasz kod był re-używalny, dlatego utwórzmy klasę, którą można użyć w różnych miejscach. Posiadanie metody main w tej klasie nam to uniemożliwia.